policyengine_us/variables/household/demographic/person/is_student.py

```diff
@@ -0,0 +1,13 @@
+from policyengine_us.model_api import *
+
+
+class is_student(Variable):
```

Comment:
does this include college and high-school

Reply:
yes, included in `is_full_time_student` variable